src/core/stream.c

```diff
@@ -659,6 +661,23 @@ QuicStreamParamSet(
         break;
     }
 
+   case QUIC_PARAM_STREAM_RELIABLE_OFFSET:
+
+        if (BufferLength != sizeof(uint64_t) || Buffer == NULL) {
+            Status = QUIC_STATUS_INVALID_PARAMETER;
+            break;
+        }
+
+        if (!Stream->Connection->State.ReliableResetStreamNegotiated ||
```

Comment:
Let's also fail if the `Stream->Flags.LocalCloseReset` is already set, indicating that the peer aborted the send before setting the offset.

Reply:
Done.